coordinator/src/main/scala/filodb/coordinator/v2/FiloDbClusterDiscovery.scala

```diff
@@ -110,7 +118,18 @@ class FiloDbClusterDiscovery(settings: FilodbSettings,
     val acc = new ShardMapper(numShards)
     val snapshots = for {
       nca <- Observable.fromIteratorUnsafe(nodeCoordActorSelections.iterator)
-      ncaRef <- Observable.fromFuture(nca.resolveOne(settings.ResolveActorTimeout).recover{case _=> ActorRef.noSender})
+      ncaRef <- Observable.fromFuture(nca.resolveOne(settings.ResolveActorTimeout)
+        .recover {
+          // recovering with ActorRef.noSender
+          case e =>
+            // log the exception we got while trying to resolve and emit metric
+            logger.error(s"[ClusterV2] Actor Resolve Failed ! actor: ${nca.toString()}", e)
+            actorResolvedFailedCounter
+              .withTag("dataset", ref.dataset)
+              .withTag("actor", nca.toString())
```

Comment:
Could these metrics suddenly explode in cardinality during a network outage?

Reply:
Thinking, so the actor and actor path should be same here for V2. since we are still using hostnames. But I agree that, we will have to update it, if we go the ip route

Reply:
Sync'd offline. Only used for V2. we only log this for failure scenarios and worst cardinality is 21k